suggest.el

```diff
@@ -296,6 +298,8 @@ See also `suggest-extra-args'.")
    ;; `-iterate' is great for building incrementing/decrementing lists.
    ;; (an alternative to `number-sequence').
    #'-iterate '(1+ 1-)
+   ;; Lists can be sorted in a variety of ways.
+   #'-sort '(< > string< string>)
```

Comment:
Aha, this is a perfect use of `suggest-extra-args`, and definitely something users might search for!